graphql/schema/wrappers.go

```diff
@@ -343,6 +344,13 @@ func (o *operation) Mutations() (ms []Mutation) {
 	return
 }
 
+func (o *operation) CacheControl() string {
+	if o.op.Directives.ForName(cacheControlDirective) == nil {
+		return ""
+	}
+	return "public,max-age=" + o.op.Directives.ForName(cacheControlDirective).Arguments[0].Value.Raw
```

Comment:
probably read the argument by name instead of arguments[0]?

Reply:
The name `maxAge` is inside the 0'th index. 
![Screenshot1603793557](https://user-images.githubusercontent.com/4118783/97287928-2c602d80-186b-11eb-9ab7-cf667bb3aa68.png)